AndrewTehan/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+ENTER_STRING = 'Enter string: '
+ERROR_MESSAGE = "String can't be blank"
+EMPTY_STRING = ''
+EXIT = 'exit!'
```

Comment:
I'd recommend being more consistent in naming, cause I don't get a point why some of the messages are called as `*_STRING`, others - `*_MESSAGE` and one of them is w/o any prefix in general